### DISCUSSION TITLE
有手动触发画布重绘的方法

### DISCUSSION THREAD

**Discussion by avrinfly** [antvis/X6#1808] [Q&A]
开启异步画布功能以后，新建一个层级介于最高层级和最低层级之间的节点，表现上，其层级最高，只有在画布重绘之后，该节点才会回到回来的层级，有人知道有什么手动触发画布的方法吗

**Comment by NewByVector** [comment]
感觉你修改 graph 的配置 sorting 就可以实现。

**Reply by avrinfly** [reply]
还是不行，我用别的方式实现了
在新建前，冻结画布，新建完后再解除冻结状态
